gems/pending/MiqContainerGroup/MiqContainerGroup.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'fs/MiqFS/modules/WebDAV'
+
+class MiqContainerGroup
```

Comment:
I don't think so. I think MiqContainerGroup corresponds to MiqVm - only it doesn't need to do much. The FS here is the WebDAV FS module.